pallets/disputes/src/tests.rs

```diff
@@ -425,6 +425,38 @@ fn extend_dispute_works_assert_state() {
     });
 }
 
+#[test]
+fn extend_dispute_too_many_disputes() {
+    new_test_ext().execute_with(|| {
+        let dispute_key = 10;
+        (0..=1000).for_each(|i| {
```

Comment:
Ok i see you are matching the `ConstU32<>` in `DisputesFinaliseOn`
A better solution would be to change the constant bound in `DisputesFinaliseOn` in favour for a `Config` type and using that instead of this staticness. 

```suggestion
        (0u32..=<T as Config::MaxDisputesPerBlock>).for_each(|i| {
```

Reply:
Done